app/javascript/packages/document-capture/components/document-capture.jsx

```diff
@@ -14,6 +14,7 @@ import { BackgroundEncryptedUploadError } from '../higher-order/with-background-
 import SuspenseErrorBoundary from './suspense-error-boundary';
 import SubmissionInterstitial from './submission-interstitial';
 import PromptOnNavigate from './prompt-on-navigate';
+import withProps from '../higher-order/withProps';
```

Comment:
we've been naming these files kebab-case, can we rename?

```suggestion
import withProps from '../higher-order/with-props';
```

Reply:
Done.